PIL/Image.py

```diff
@@ -2223,6 +2317,7 @@ def _check_size(size):
 
 
 def new(mode, size, color=0):
+    # type: (Mode, Size, Optional[Union[int, float, Tuple, Text]]) -> Image
```

Comment:
No, not all colors are going to take the text equivalents, though the draw, fill, (and maybe paste) are going to. The pyaccess and putpixel are much more constrained to the underlying type of the image, and getpixel is going to be exactly related to the underlying pixel type.